src/content/docs/infrastructure/host-integrations/host-integrations-list/active-directory-integration.mdx

```diff
@@ -26,50 +23,71 @@ Built with our infrastructure agent, the Active Directory integration gives you
 After setting up the Active Directory integration with New Relic, see your data in a dashboard like this, right out of the box. 
 </figcaption> 
 
+To get the most out of this page, select the installation method that fits your environment. You need a [New Relic account](https://newrelic.com/signup) before starting the installation process.
```

Comment:
Looks like this is just a one path install, so you could probably delete this text